### ISSUE TITLE
Fluctuating Validation Performance

### ISSUE THREAD

**Issue by sondosaabed** [sondosaabed/Iris-of-eyes-recognition#1]
- [ ] Work on stabilizing training before the 20th epoch, the validation is fluctuating

**Comment by sondosaabed** [comment]
It's not a problem as long as it's following an up pattern in case of accuracy and a down pattern in terms of loss. 
This might be due to the split rule of 10% for the validation split or due to the disharmony between batch normalization and dropout regularization.

**Comment by sondosaabed** [comment]
I have changed the LR the learning curve shows a clearer convergence, still there is fluctuating in validation, my guess this is due to variability in the dataset ? will look further for a smoother curve
